providers/base/units/audio/jobs.pxu

```diff
@@ -374,7 +374,7 @@ estimated_duration: 1.0
 imports: from com.canonical.plainbox import manifest
 requires:
   package.name == 'pulseaudio-utils'
-  manifest.has_audio_playback == 'True'
+  audio_card.Playback == 'supported'
```

Comment:
This change is to avoid using manifest(more automated), and Pierre suggests using `supported` or `unsupported`, so QA could easier to understand the meaning without confusion.